jest.config.json

```diff
@@ -14,5 +14,9 @@
             "functions": 88,
             "lines": 88
         }
-    }
+    },
+    "coveragePathIgnorePatterns": [
+        "<rootDir>/node_modules/",
+        "<rootDir>/utils/testUtils.js"
+    ]
```

Comment:
Great! We can also ignore files like mockData.js, testServer.js. Do you want to raise a PR to the orginal template to add these?